lib/action_controller/serialization.rb

```diff
@@ -6,7 +6,7 @@ module Serialization
 
     include ActionController::Renderers
 
-    ADAPTER_OPTION_KEYS = [:include, :fields, :root, :adapter]
+    ADAPTER_OPTION_KEYS = [:include, :fields, :adapter]
```

Comment:
Yeah, we realized we forgot about the README, but I'm already updating it.
I'm glad it didn't broke #954 :smile: